Websites/webkit.org/demos/webgpu/scripts/hello-triangle.js

```diff
@@ -30,17 +30,17 @@ async function helloTriangle() {
 /*
     FIXME: Use WGSL once compiler is brought up
     const wgslSource = `
-                     @vertex fn main(@builtin(vertex_index) VertexIndex: u32) -> @builtin(position) vec4<f32>
+                     @vertex fn vsmain(@builtin(vertex_index) VertexIndex: u32) -> @builtin(position) vec4<f32>
                      {
-                         var pos = array<vec2<f32>, 3>(
+                         var pos: array<vec2<f32>, 3> = array<vec2<f32>, 3>(
                              vec2<f32>( 0.0,  0.5),
                              vec2<f32>(-0.5, -0.5),
                              vec2<f32>( 0.5, -0.5)
                          );
                          return vec4<f32>(pos[VertexIndex], 0.0, 1.0);
                      }
 
-                     @fragment fn main() -> @location(0) vec4<f32>
+                     @fragment fn fsmain() -> @location(0) vec4<f32>
                      {
```

Comment:
What would it take to make real tests, rather than just these demo files? I think we should either A) use existing tests from the CTS from upstream, or B) write our own real tests and contribute them to the upstream CTS.

Reply:
I'll investigate that, I honestly haven't started running the real tests yet, I was just playing around with the demo for now, but I agree we should start testing it properly.

Reply:
I agree we should do this too but it is probably outside the scope of this PR.

@litherum do you know who is responsible for maintaining [the CTS](https://github.com/gpuweb/cts)? Is that any of the gpuweb contributors? I think its non-functional at the moment

Reply:
We have a snapshot of the CTS already in WebKit that runs on every commit.

Yes, the maintainers are the WebGPU contributors.